lineman/app/components/change_password_form/change_password_form.coffee

```diff
@@ -0,0 +1,17 @@
+angular.module('loomioApp').factory 'ChangePasswordForm', ->
+  templateUrl: 'generated/components/change_password_form/change_password_form.html'
+  controller: ($scope, $rootScope, CurrentUser, Records, FlashService) ->
+    $scope.user = CurrentUser.clone()
+
+    $scope.submit = ->
+      $scope.isDisabled = true
+      Records.users.changePassword($scope.user).then ->
+        $scope.isDisabled = false
+        FlashService.success('profile_page.messages.password_changed')
+        $scope.$close()
+      , (response) ->
+        $scope.isDisabled = false
+        if response.status == 422
```

Comment:
Yea I think it's pretty simple. I like the submit, isDisabled, close simplicity.

I think setErrors should be done within the records_interface function with chained promises. (like save() on Model)

With all the network calls moving to the record_interface I'm thinking we should consider moving base_model#save to the records_interface then we wouldn't need restful_client in the model at all. Then we separate the network stuff from the local stuff (find, where) completely?